moveit_ros/planning_interface/moveit_cpp/include/moveit/moveit_cpp/planning_component.h

```diff
@@ -108,10 +108,21 @@ class PlanningComponent
   {
     std::string planner_id;
     std::string planning_pipeline;
-    size_t planning_attempts;
+    int planning_attempts;
```

Comment:
I changed it because the type of `num_planning_attempts` in `moveit_msgs::MotionPlanRequest` is `int` and `NodeHandle::getParam` don't have an overload for `size_t`